movieApp/WebAPI/Movie_WebAPI/Movie_WebAPI/Controllers/MovieController.cs

```diff
@@ -0,0 +1,158 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Movie_WebAPI.Helpers;
+using Movie_WebAPI.Services;
+using MovieApp.Models;
+using MovieApp.Services;
+
+namespace Movie_WebAPI.Controllers
+{
+    public class MovieController : Controller
+    {
+        private readonly MovieService _movieService;
+        private readonly LogService _logService;
+
+        public MovieController(MovieService movieService, LogService logService)
+        {
+            _movieService = movieService;
+            _logService = logService;
+        }
+
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+        [Route("api/addMovie")]
+        [HttpPost]
+        public ActionResult<string> AddMovie([FromBody] Movie movie)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState);
+            }
```

Comment:
create a validator which checks the model state, this way you will avoid a lot of code duplication